addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -242,19 +242,12 @@ PortalChatter.include({
     _onClickStarDomain: function (ev) {
         var $tr = this.$(ev.currentTarget);
         var num = $tr.data('star');
-        if ($tr.css('opacity') === '1') {
-            this.set('rating_value', num);
-            this.$('.o_website_rating_select').css({
-                'opacity': 0.5,
-            });
-            this.$('.o_website_rating_select_text[data-star="' + num + '"]').css({
-                'visibility': 'visible',
-                'opacity': 1,
-            });
-            this.$('.o_website_rating_select[data-star="' + num + '"]').css({
-                'opacity': 1,
-            });
-        }
+        this.set('rating_value', num);
+        // The previous statement will trigger a RPC and a re-rendering if the rating_value has changed
+        // so the following lines are only useful to update the interface during the RPC call to get a quick interface update
+        this.$('.o_website_rating_progress_table').addClass('o_website_rating_progress_table_has_selection');
+        this.$('.o_website_rating_select').removeClass('o_website_rating_table_row_selected');
+        this.$('.o_website_rating_select[data-star="' + num + '"]').addClass('o_website_rating_table_row_selected');
```

Comment:
Thanks for the propositions and the discussion with Luc. 
Here is a summary (don't hesitate if I missed something) that I have implemented:
- add a spinner on the message list while loading but only when the connection is slow (more than 500 millisecond to respond to the RPC call)
- remove the code that update the interface so that the user get a quick feedback (most connection are fast and we have now the spinner)
- add some minimal height to avoid to see the footer moving when switching from course to review tab or changing the star filter (which change the number of messages displayed). Note that I haven't found (probably missed it) a similar feature in event or appointment so I have just used a minimal height based on a percentage of the view port to push the footer outside the screen.